backend/libs/authorization.js

```diff
@@ -287,6 +298,28 @@ exports.authorizeAddNewUser = function(req, res, next) {
     }
 };
 
+exports.authorizeGetListByEmail = function(req, res, next) {
+    var currentUser = req.user;
+    var email = utils.getEmailParam(req);
+
+    if(currentUser.email === email) {
+        next();
+    } else {
+        send403(res);
+    }
+};
+
+exports.authorizeChangeAccount = function(req, res, next) {
+  var currentUser = req.user;
+  var email = req.body.email;
+
+  if(currentUser.email === email && req.body.external && currentUser.external.id === req.body.external.id) {
```

Comment:
Do you send all customer in body? I can send any body. We need to check it via DB